jobs/jira_burndown.rb

```diff
@@ -74,19 +75,21 @@ def sprintEnd
   def jsonChangesByDateTime
     @json["changes"].find_all {|key, value|
       value.find_all {|subEntry|
-        ! subEntry["timeC"].nil?
+        ! subEntry["statC"].nil? ||
+        ! subEntry["column"].nil? 
       }.length > 0
     }.map{ |key, value|
       timeChanges = value.find_all { |valueEntry|
-        ! valueEntry["timeC"].nil?
+        ! valueEntry["statC"].nil? ||
+        ! valueEntry["column"].nil?
       }
-      [Time.at(key.to_i / 1000), timeChanges]
+      [Time.at(key.to_f / 1000), timeChanges]
```

Comment:
From what I can tell, the duration fields are all non-floating point numbers, is this different for the story-point values?